quandl/utils/request_type_util.py

```diff
@@ -0,0 +1,20 @@
+try:
+    from urllib import urlencode
+except ImportError:
+    from urllib.parse import urlencode
+
+from quandl.api_config import ApiConfig
+
+
+class RequestType(object):
+    MAX_URL_LENGTH_FOR_GET = 8000
```

Comment:
do you think if it might be useful to add a comment with a link for the 8000 length?

Reply:
Yes, but I think a comment without a link will have to make do. The 8000 number was determined by researching multiple sources such as stack overflow and our nginx setup so one link wouldnt do it justice.

Reply:
make sense